docs/spec/protocol.rst

```diff
@@ -51,9 +50,9 @@ at the end of the list. Here is a simple example::
       def close(self) -> None:
           ...
 
-Now if one defines a class ``Resource`` with a ``close()`` method that has
-a compatible signature, it would implicitly be a subtype of
-``SupportsClose``, since the structural subtyping is used for
+Now if one defines a class ``Resource`` with a ``close()`` method whose type
+signature is assignable to ``SupportsClose.close``, it would implicitly be
+assignable to ``SupportsClose``, since structural assignability is used for
```

Comment:
I think this makes it clearer that you're introducing a new term:

```suggestion
assignable to ``SupportsClose``, since "structural assignability" is used for
```